apps/accounts/views.py

```diff
@@ -16,6 +21,12 @@
 )
 
 from .permissions import HasVerifiedEmail
+from base.utils import send_email
+from challenges.utils import get_challenge_model
+from hosts.utils import is_user_a_host_of_challenge
+from hosts.models import ChallengeHost
+
+logger = logging.getLogger(__name__)
```

Comment:
You are adding all these imports in this file but not using them anywhere in this file but didn't add any method that uses these imports. Am I missing something here?